Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/CalendarViewSamplePage.xaml

```diff
@@ -0,0 +1,99 @@
+﻿<Page x:Class="Uno.Gallery.Views.SamplePages.CalendarViewSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout>
+			<local:SamplePageLayout.MaterialTemplate>
+				<DataTemplate>
+					<StackPanel Spacing="20">
+						<smtx:XamlDisplay UniqueKey="CalendarViewSamplePage_Material_CalendarView"
+										  smtx:XamlDisplayExtensions.Header="CalendarView">
```

Comment:
Might be better to use `XamlDisplayBelowStyle`, so that the button doesnt go offscreen on small device.
The calendarview itself is close to fullscreen width.
\---
Same for the cupertino one below.

Reply:
I don't see that style name anywhere in Gallery

Reply:
just grab it from uno.themes